File: VB/GrowthWare.Framework/Model/Profiles/Interfaces/IMessageProfile.vb
Namespace Model.Profiles.Interfaces
    Public Interface IMessageProfile
        Inherits IMProfile

        Property Body() As String
        Property Title() As String
        Property FormatAsHTML() As Boolean
        Sub FormatBody()
        Function GetTags(ByVal seporator As String) As String
    End Interface
End Namespace

File: GrowthWare_CS/GrowthWareFramework/Interfaces/IMessageProfile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GrowthWare.Framework.Interfaces
{
	public interface IMessageProfile
	{
		string Body
		{
			get;
			set;
		}
		string Title
		{
			get;
			set;
		}
		bool FormatAsHTML
		{
			get;
			set;
		}
		void FormatBody();
		string Tags(string Seporator);
	}

}
